source/indexes.txt

```diff
@@ -173,6 +173,18 @@ Limitations
 - The :guilabel:`Indexes` tab is not available if you are connected 
   to a :atlas:`Data Lake </data-lake>`. 
 
+- You can only manage Atlas Search indexes in Compass if your deployment is
```

Comment:
(nit -- [positive statements](https://www.mongodb.com/docs/meta/style-guide/writing/use-positive-statements/))
You could omit "only" here if you like. The sentence will mean the same thing, but is more positive.

Reply:
Thanks for the edit, just fixed!